samigo/samigo-app/pom.xml

```diff
@@ -276,6 +276,16 @@
             <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
             <version>${sakai.jakarta.jstl-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20210307</version>
```

Comment:
Stick to the Sakai version, it is defined in Master, you probably don't need to include the version here.